api/rds.proto

```diff
@@ -279,6 +282,12 @@ message RouteMatch {
   // the request with the same values (or based on presence if the value field
   // is not in the config).
   repeated HeaderMatcher headers = 6;
+
+  // [#not-implemented-hide:] Specifies a set of URL query parameters on which the route should
+  // match. The router will check the query string from the *path* header against all the specified
+  // query args. If the number of specified query args is nonzero, they all must match the *path*
+  // header's query string in order or a match to occur.
```

Comment:
This might be a good place to add examples that make the use case clear. And possibly how this interacts with header match/path match etc.